python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/management-interfaces.j2

```diff
@@ -35,6 +35,36 @@
 {%         set int_type = management_interface.type | arista.avd.default('oob') %}
 | {{ management_interface.name }} | {{ description }} | {{ int_type }} | {{ vrf }} | {{ ipv6 }} | {{ ipv6_gateway }} |
 {%     endfor %}
+{%     if management_interfaces | arista.avd.default([]) | selectattr('redundancy','arista.avd.defined') | list %}
+
+##### Interface Redundancy
+{%         for mgmt_intf in management_interfaces | selectattr('redundancy', 'arista.avd.defined') | arista.avd.natural_sort('name') %}
+
+###### {{mgmt_intf.name}}
+
+| Settings | Value |
+| -------- | ----- |
+{%             if mgmt_intf.redundancy.fallback_delay is arista.avd.defined %}
+| Fallback Delay | {{ mgmt_intf.redundancy.fallback_delay }} |
+{%             endif %}
+{%             if mgmt_intf.redundancy.monitor.link_state is arista.avd.defined %}
+| Monitor Link State | {{ mgmt_intf.redundancy.monitor.link_state }} |
+{%             endif %}
+{%             if mgmt_intf.redundancy.monitor.neighbor is arista.avd.defined %}
```

Comment:
should probably be elif like in EOS template? So that both cannot be rendered at the same time